src/app/screens/Settings.tsx

```diff
@@ -56,6 +57,21 @@ function Settings() {
           )}
         </Setting>
 
+        <Setting
+          title="Legacy LNURL Auth"
+          subtitle="Login with the deprecated LNURL Auth method"
```

Comment:
maybe more details for the user: 

We have made compatibility changes to the signing with LNDhub and LNbits accounts. If you used LNURL-auth logins with those accounts you can enable the deprecated signing method. (not recommended)